packages/btcindexer/src/cf-storage.ts

```diff
@@ -213,7 +213,13 @@ export class CFStorage implements Storage {
 
 		const statements = updates.map((p) => {
```

Comment:
```suggestion
		const statements = updates.map((u) => {
```
why `p`? `u` makes more sense